### ISSUE TITLE
Installer silently fails to install Rainway

### ISSUE THREAD

**Issue by shirbr510** [RainwayApp/bug-tracker#273]
# Installer Doesn't seem to actually work

I've been trying to use rainway since version 0.2.54 and it never worked. not even once.

I had a previous bug openned #217  and it was closed so I assume "hey, they might have actually fixed things!"

so I have tried installing version 1.0.4, and again, installer is starting and.... nothing happens

I was assuming "hey, maybe it did something, maybe it's running a service behind the scenes", but nope, nothing running.

## Steps

- Download client from https://rainway.com/download-success/?from=download (v1.0.4.0)
- click install
- click "yes" on the admin prompt
- Nothing seems to work

## What happens

- Nothing seems to work

## What is expected

- That it will work somehow

## Host

 - OS: Windows 10 Home
 - Version: Version 10.0.17763 Build 17763
 
## Browser

 - OS: Windows 10 Home
 - Version: Version 10.0.17763 Build 17763
 - Browser: Chrome
 - Browser Version: 71.0.3578.98

## Rainway Version
1.0.4 - 09/02/19
1.0.6 - 18/02/19
 
## Logs
None that I could find

**Comment by shirbr510** [comment]
p.s, @FrobtheBuilder, can you please explain at least why was my ticket closed / not actually treated for almost a year?

**Comment by lynn** [comment]
Hey @shirbr510, thank you for being patient with us and giving Rainway another shot!

We've heard from multiple users that they're having with the latest installer, and we're looking into the issue this week. I'll ping you here when we resolve it and push a version with a fix.

(Sorry your last ticket was closed without proper communication on our end. The 1.0 release is such a drastic change to all parts of Rainway, including the installer, that it wouldn't have been useful to keep around issues from _before_ then anymore, so we've been doing some cleaning up. The upside of this is: chances are pretty good your issue was resolved! We hope you'll let us know once we get you up and running. ✨ )

**Comment by shirbr510** [comment]
It's alright and I'm looking forward to hear from you :)
as you said, communication is key and I'm glad that you understand that

let me know if you need any additional information or anything
(by chance, I'm a software developer so let me know if I can come in handy in getting my hands dirty too 😄 )

**Comment by sakshamgoyal01** [comment]
I have the same issue. follow my thread too. there are a few possible solutions there.

**Comment by shirbr510** [comment]
@sakshamgoyal01 thank you :)
next time, please add a link to your issue so it will be easier to find it 😄  ( #272  )

**Comment by shirbr510** [comment]
followed the steps on #272 and it seems like the radar service doesn't even get installed properly on my computer
![image](https://user-images.githubusercontent.com/6445074/52920489-ab226a80-3315-11e9-89fa-46cd205d57f3.png)

**Comment by shirbr510** [comment]
@lynn I have tried installing 1.0.6 and the problem still exists

**Comment by naldo29** [comment]
im having the same issie, if one of the devs want to check my pc for the issue, then your all welcome.

**Comment by AnthonyM** [comment]
Could someone experiencing this issue please try 1.0.9.0

There are some installer fixes which may help.

**Comment by BetaLeaf** [comment]
What command line flags are you using? Steps to reproduce?

**Comment by shirbr510** [comment]
here we are, a year later, and I'm trying your app again

problem seems to still remain @ Dashboard version 1.0.51

**Comment by xorinzor** [comment]
Just copying the comment from Andrew on Discord here: 

"when you run the bootstrapper.exe it creates a bootstrapper.log
send that to me"

Can you upload the log here too?

**Comment by shirbr510** [comment]
hey @xorinzor , and sorry for the late response.
these are the contents of the bootstrapper log file:

> [bootstrapper][INFO] Failed or not need to detach session.
[bootstrapper][INFO] Rainway installed: false
[bootstrapper][INFO] update type: Install
[bootstrapper][INFO] checking system compatibility.
[bootstrapper::rainway][INFO] The current .NET Framework version is: 528040
[bootstrapper::rainway][INFO] current system is compatible.
[bootstrapper][INFO] user branch: Stable
[quiche::updater][INFO] pulling the latest release for the Stable branch
[quiche::updater][INFO] download background thread started
[quiche::net::http][INFO] starting download of https://releases.rainway.com/quiche/Stable/1.0.51/installer.exe (6990480 bytes) to C:\Users\Shir\AppData\Local\Temp\74AC56FD003A9F5BAC505B6B3422BEE08A37CDB2197F791D2C16347C18BC2265.exe.
[quiche::net::http][INFO] downloaded 6990480 bytes.
[quiche::updater][INFO] unblocking C:\Users\Shir\AppData\Local\Temp\74AC56FD003A9F5BAC505B6B3422BEE08A37CDB2197F791D2C16347C18BC2265.exe
[quiche::updater][INFO] download background thread finished.
[quiche::updater][INFO] hashing C:\Users\Shir\AppData\Local\Temp\74AC56FD003A9F5BAC505B6B3422BEE08A37CDB2197F791D2C16347C18BC2265.exe
[quiche::updater][INFO] finished hashing C:\Users\Shir\AppData\Local\Temp\74AC56FD003A9F5BAC505B6B3422BEE08A37CDB2197F791D2C16347C18BC2265.exe
[quiche::updater][INFO] running C:\Users\Shir\AppData\Local\Temp\74AC56FD003A9F5BAC505B6B3422BEE08A37CDB2197F791D2C16347C18BC2265.exe
[quiche::updater][INFO] 

it appears as if the installation was successful, but then when I'm trying to start rainway, nothing happens

**Comment by AnimusNefarius** [comment]
Version 1.0.54 and I am having the same problem. 
I have tried several things from a few similar threads.
Nothing seems to work,
My bootstrap log currently looks like this:
[bootstrapper][INFO] Failed or not need to detach session.
[bootstrapper][INFO] Rainway installed: true
[bootstrapper][INFO] update type: Patch
[bootstrapper][INFO] user branch: Stable
[quiche::updater][INFO] pulling the latest release for the Stable branch
[bootstrapper][INFO] validating Rainway installation.
[bootstrapper][INFO] Rainway is not outdated, starting.
[bootstrapper::rainway][INFO] Failed or not need to detach session.
[bootstrapper::rainway][INFO] Rainway service started: false

**Comment by shirbr510** [comment]
@xorinzor any response please?

**Comment by xorinzor** [comment]
@shirbr510 Unfortunately there isn't anything in there that would help me to diagnose the issue.
Either way, I mostly asked you to add the log file for completeness of the issue, I'm not part of the Rainway team, just a fellow user who perhaps could've helped you.

Windows (installers) aren't really my area of expertise. Hopefully someone from the rainway team will help you.

**Comment by shirbr510** [comment]
@xorinzor oh, sorry 😅
I really wish that they would handle the issue...
They do seem more responsive @ their discord so I'll probably repost the issue and hope for the best

**Comment by shirbr510** [comment]
Version 1.0.88 finally solved my struggle 🎉 

than kyou so much 🙏
